src/main/java/frc/robot/SimulationManager.java

```diff
@@ -0,0 +1,56 @@
+package frc.robot;
+
+import edu.wpi.first.math.geometry.Pose3d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Rotation3d;
+import edu.wpi.first.math.geometry.Translation3d;
+import org.littletonrobotics.junction.Logger;
+
+public class SimulationManager {
+
+	private final String logPath;
+	private final Robot robot;
+
+	public SimulationManager(String logPath, Robot robot) {
+		this.logPath = logPath;
+		this.robot = robot;
+	}
+
+	public void logPoses() {
+		logIntakePosition3d();
+		logTurretPosition3d();
+		logHoodPosition3d();
+	}
+
+	private void logIntakePosition3d() {
+		Logger.recordOutput(logPath + "/Intake/Position", getIntakePosition3d(robot.getFourBar().getPosition()));
```

Comment:
```suggestion
		Logger.recordOutput(logPath + "/Intake, getIntakePosition3d(robot.getFourBar().getPosition()));
```
and in all the others